sycl/include/sycl/device_aspect_macros.hpp

```diff
@@ -740,3 +745,8 @@
 //__SYCL_ASPECT(ext_oneapi_bindless_sampled_image_fetch_3d, 72)
 #define __SYCL_ANY_DEVICE_HAS_ext_oneapi_bindless_sampled_image_fetch_3d__ 0
 #endif
+
+#ifndef __SYCL_ANY_DEVICE_HAS_ext_oneapi_queue_profiling_tag__
+// __SYCL_ASPECT(ext_oneapi_queue_profiling_tag, 67)
```

Comment:
Where does the 67 value come from? It's defined as 73 in aspects.def

Reply:
You're absolutely right. This has been fixed.